preprocessing/xml_loader.py

```diff
@@ -94,7 +107,7 @@ def set_contracted_hours_for_employee(employee, schedule_row):
 def get_days_with_demand(root):
     demands = get_demand_definitions(root)
     days_with_demand = {}
-    for day in root.findall("Demands/DayDemandList/DayDemand"):
+    for day in root.findall('Demands/DayDemandList/DayDemand'):
```

Comment:
I hope this change is only because of merging, and not some string preference for single quotes? My automatic formatter always changes to double quotes you see.

Reply:
Most likely because of merge. I have not changed it to singles, but must be from my initial implementation